docs/examples/rx/03_listen_to_balance_change/index.js

```diff
@@ -30,7 +30,12 @@ async function main () {
         return;
       }
 
-      const change = balance[1].sub(balance[0]);
+      let change;
```

Comment:
there's a function for this kind of thing: `Math.abs()`

Reply:
I will check..if balance type has this.. cause all these calculation are being performed on balance type.. If it were a number then I could use Math.abs()